src/main/resources/application.conf

```diff
@@ -5,6 +5,9 @@ file {
   }
   nynta_path = "src/main/resources/nynta.csv"
   population_path = "src/main/resources/population.csv"
+  split_dataset_base_folder = "/home/uladzimir/split_dataset"
```

Comment:
Seems to be OS specific path, could you please make it more abstract? It will not work for Windows for example, use System class for retrieving user home folder path. Please see comments below

Reply:
But it's config file. When we will use it in Windows we just replace path. Or it's bad?

Reply:
Do you have two different files for Linux/Windows? If no, then your potential Windows user will be unhappy after failed running of your app. Your app should run in ""one-click" fashion without extra efforts and diving into code. Leave in configs paths relative to user.home and prepend user.home in the code. So you'll improve portability of your app with only one config file. Does it make sense?

Reply:
Please, take a look typesafe [docs](https://github.com/lightbend/config#features-of-hocon)

> substitutions fall back to environment variables if they don't resolve in the config itself, so ${HOME} would work as you expect. Also, most configs have system properties merged in so you could use ${user.home}.

Reply:
Done